src/VisualStudio/Core/Def/Implementation/Utilities/AutomationDelegatingListView.cs

```diff
@@ -88,6 +88,7 @@ internal class AutomationDelegatingListViewItemAutomationPeer : ListBoxItemWrapp
         public AutomationDelegatingListViewItemAutomationPeer(AutomationDelegatingListViewItem listViewItem)
             : base(listViewItem) 
         {
+            var x = this.GetChildren();
```

Comment:
?

Reply:
This is used to debug previously, should be removed.